python/paddle/fluid/tests/unittests/dygraph_to_static/test_ifelse.py

```diff
@@ -116,6 +116,30 @@ def setUp(self):
         self.dyfunc = dyfunc_ifExp_with_while
 
 
+def dyfunc_ifExp_with_while2(x):
```

Comment:
Why is it named `dyfunc_ifExp_with_while2`? I didn't see `while` in this test

Reply:
I forgot to rename it. I will add commit to fix this.

Reply:
After offline discussion, we will merge this at first due to CI time, then rename to `dyfunc_ifExp` immediately at https://github.com/PaddlePaddle/Paddle/pull/23478